src/Forms/FileField.php

```diff
@@ -172,7 +172,7 @@ public function saveInto(DataObjectInterface $record)
         }
     }
 
-    public function Value()
+    public function getValue(): mixed
     {
         return isset($_FILES[$this->getName()]) ? $_FILES[$this->getName()] : null;
     }
```

Comment:
Can you please explain why this one isn't `getFormattedValue()`? You're probably right, I just don't have the context for it.
Also affects `FileUploadReceiver` by the looks of it.

Reply:
It derives its value from the the $_FILES superglobal i.e. the raw filename that was uploaded by the user. It's not something that would be changed by localisation

Reply:
I don't think localisation is relevant here but yup that does sound raw-ish